src/creds/CredentialSignerVerifier.js

```diff
@@ -0,0 +1,60 @@
+const _ = require('lodash');
```

Comment:
instead of importing lodash as _, can't we start requiring the specific functions and stop using the _.

sample

const { isEmpty } = require('lodash')

isEmpty() instead of _.isEmpty

This _. pollutes so much the code

Reply:
IMO _.isEmpty is more clear since show it the `isEmpty` funtiction is a loadash function since importing it as "_" is a defacto standard since way back.